packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -293,26 +148,14 @@ class ValidatorsList extends React.PureComponent<Props, State> {
       return a > b ? 1 : -1
     }
 
-    return (data || [])
-      .sort((a, b) => compare(dAccessor(a), dAccessor(b)))
-      .sort((a, b) => dir * compare(accessor(a), accessor(b)))
-      .sort((a, b) => this.isPinned(b) - this.isPinned(a))
-  }
-
-  isPinned({ address }: any) {
-    const list = (localStorage.getItem(localStoragePinnedKey) || '').split(',') || []
-    return +list.includes(address)
-  }
+    this.cachedCleanData = (data || []).sort((a, b) => dir * compare(accessor(a), accessor(b)))
 
-  onPinned() {
-    this.setState({ update: Math.random() } as any)
+    return (data || []).sort((a, b) => dir * compare(accessor(a), accessor(b)))
```

Comment:
I really really wanted to simplify these two lines, but no matter what I try, it causes a weird bug to occur.  Really wanted to either return data or assign to `this.cachesCleanData` (instead of doing both) or do something like
```
const sorted = (data || []).sort((a, b) => dir * compare(accessor(a), accessor(b)))
this.cachesCleanData = sorted
return sorted
```
but all of those cause a bug with the sort 😒